shared/xccdf/system/accounts/physical.xml

```diff
@@ -253,6 +253,52 @@ must be automated as a component of machine provisioning, or followed manually a
 cis="1.4.2" />
 </Rule>
 
+<Rule id="bootloader_no_removeable_media" severity="medium" prodtype="rhel7">
+<title>Boat Loader Is Not Installed On Removeable Media</title>
+<description>
+The system must not allow removable media to be used as the boot loader.
+Remove alternate methods of booting the system from removable media.
+</description>
+<ocil clause="it is not">
+To verify the system is not configured to use a boot loader on removable media,
+run the following command:
+<pre>$ sudo grep "set root='hd0" /boot/grub2/grub.cfg</pre>
+The output should return something similar to:
+<pre>set root='hd0,msdos1'</pre>
+</ocil>
+<rationale>
+Malicious users with removable boot media can gain access to a system
+configured to use removable media as the boot loader.
+</rationale>
+<platform idref="cpe:/a:machine" />
+<ident cce="80517-6"/>
+<oval id="bootloader_no_removeable_media"/>
+<ref disa="1814" nist="" srg="SRG-OS-000364-GPOS-00151" stigid="021700"/>
+</Rule>
+
+<Rule id="bootloader_uefi_no_removeable_media" severity="medium" prodtype="rhel7">
+<title>UEFI Boat Loader Is Not Installed On Removeable Media</title>
+<description>
+The system must not allow removable media to be used as the boot loader.
+Remove alternate methods of booting the system from removable media.
+</description>
+<ocil clause="it is not">
+To verify the system is not configured to use a boot loader on removable media,
+run the following command:
+<pre>$ sudo grep "set root='hd0" /boot/efi/EFI/redhat/grub.cfg</pre>
+The output should return something similar to:
+<pre>set root='hd0,msdos1'</pre>
```

Comment:
Example of what a removable media entry would be?

Reply:
@shawndwells added this:

```<tt>usb0</tt>, <tt>cd</tt>, <tt>fd0</tt>, etc. are some examples of removeable
media which should not exist in the line:
<pre>set root='hd0,msdos1'</pre>```